pandas/tests/arrays/test_integer.py

```diff
@@ -339,6 +339,13 @@ def test_error(self, data, all_arithmetic_operators):
         with pytest.raises(NotImplementedError):
             opa(np.arange(len(s)).reshape(-1, len(s)))
 
+    def test_divide_by_zero(self):
+        # https://github.com/pandas-dev/pandas/issues/27398
+        a = pd.array([0, 1, -1, None], dtype="Int64")
+        result = a / 0
```

Comment:
Can you parametrize this over different versions of zero, e.g. `0`, `-0`, `0.0`, `-0.0`?

Strangely, it looks like the `-0.0` case is inconsistent with the others, as the signs of `np.inf` are flipped:
```python
In [1]: import numpy as np; import pandas as pd; pd.__version__
Out[1]: '0.26.0.dev0+1272.g5374d8b41'

In [2]: a = pd.array([0, 1, -1, None], dtype="Int64")

In [3]: a / 0
Out[3]: array([ nan,  inf, -inf,  nan])

In [4]: a / -0
Out[4]: array([ nan,  inf, -inf,  nan])

In [5]: a / 0.0
Out[5]: array([ nan,  inf, -inf,  nan])

In [6]: a / -0.0
Out[6]: array([ nan, -inf,  inf,  nan])
```

Reply:
Python apparently converts defines `-0` (or `operator.__neg__(0)`) as 0. So `a / -0` is equivalent to `a / 0`.

Added tests for -0.0 and 0.0.

Reply:
ah, yeah, for ints -0 and 0 have the same binary representation, but since floats have a bit dedicated to sign 0.0 and -0.0 are distinct